packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -550,7 +559,12 @@ class Auth {
             ? () => helpers.loginGuestUserPrivate(...guestPrivateArgs)
             : () => helpers.loginGuestUser(...guestPublicArgs)
 
-        return await this.queueRequest(callback, isGuest)
+        try {
+            return await this.queueRequest(callback, isGuest)
+        } catch (error) {
+            await this.logTokenRequestError(error as Error)
+            throw error
```

Comment:
Because `error.response` has been consumed with `json()`, when we throw an error here, the caller can no longer read what's inside `error.response` anymore.

Reply:
Oh, inspired by what's seen in your other PR: rather than reusing the same error, let's create a _new_ error (with `HTTPError`) and throw that one instead.

Reply:
I've updated this to throw a new Error (since commerce-sdk-react currently does not have access to HTTPError).